### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by luco5826**

### CODE REVIEW

File: src/util/StringUtils.cpp [hunk 1]
```diff
@@ -5,12 +5,8 @@
 #include <string.h>
 
 string StringUtils::toLowerCase(string input)
-{
-	char* lower = g_utf8_strdown(input.c_str(), input.size());
-	string strLower = lower;
-	g_free(lower);
-
-	return strLower;
+{
```
Comment (andreasb242) [inline src/util/StringUtils.cpp]: This produces a memory leak

https://developer.gnome.org/glib/stable/glib-Unicode-Manipulation.html#g-utf8-strdown

Comment (andreasb242) [reply src/util/StringUtils.cpp]: Please be really carefull with Memory stuff, such bugs are really hard to find

Comment (luco5826) [reply src/util/StringUtils.cpp]: Ok, so it needs to be like before this change?

Comment (andreasb242) [reply src/util/StringUtils.cpp]: Yes.

**Pull request closed without merging**